bot.ts

```
@@ -28,7 +28,7 @@
 import wikisData from "./data/wikis.json";
 import templatesData from "./data/templates.json";
 import filesData from "./data/files.json";
-import { MwnError } from "mwn/build/error";
+import { MwnError, MwnMissingPageError } from "mwn/build/error";
```

ESLint

[error] 'MwnError' is defined but never used.

(@typescript-eslint/no-unused-vars)